libs/blocks/review/review.js

```diff
@@ -115,15 +116,16 @@ const removeMetaDataElements = (el) => {
 };
 
 export default async function init(el) {
-  const { miloLibs, codeRoot } = getConfig();
+  const { miloLibs, codeRoot, env } = getConfig();
   const base = miloLibs || codeRoot;
 
   loadStyle(`${base}/ui/page/page.css`);
   const metaData = getMetaData(el);
   const strings = getStrings(metaData);
+  strings.postUrl = strings.postUrl ? checkPostUrl(strings.postUrl, env) : strings.postUrl;
```

Comment:
I'm trying to understand this line: if the `strings.postUrl` is undefined or empty then you are assigning it back. Do we even need to init the application in this case? I see that `getReviewPath` will handle the error, but don't think we need to go that far if we know the URL is not valid.